bazel/external/wee8.genrule_cmd

```diff
@@ -33,6 +33,8 @@ export NM=$${NM:-nm}
 if [[ $${ENVOY_ASAN-} == "1" ]]; then
```

Comment:
@lizan do we want to piggy-back on `ENVOY_ASAN` here, or should we add new `ENVOY_UBSAN_VPTR` to differentiate between the two?

Reply:
yeah that might be better, you need add that def in clang_setup.sh as well.

Reply:
Done. All tests passed on the CI and I verified locally that this change fixes build with libc++ and `--sanitize=vptr,function`.